.github/PENDING_REVIEW_NOTIFICATION_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Hi {{ username }},
+
+It looks like you're assigned to this PR, but haven't taken any action for at least 2 days:
+{{ pr_list }}
+
+Please review and unassign yourself from the pending PRs as soon as possible, and reply to this thread once all the PRs are reviewed.
```

Comment:
```suggestion
Please review and unassign yourself from the pending PRs as soon as possible.
```

Reply:
@U8NWXD Do you agree that we should make this change since we are not expecting reviewers to reply?

Reply:
Yep, I agree! Let me know when you want me to merge this PR

Reply:
Let's merge this today